src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int value = random.nextInt(100);
```

Comment:
as described in common mistakes, let's avoid using magic numbers

Reply:
not fixed. 
also  variable `value` is redundant, used only one times. so  `random.nextInt()` can be used directly in place where it is called